warehouse/app_test.go

```diff
@@ -107,6 +108,12 @@ func TestApp(t *testing.T) {
 					pgResource, err := postgres.Setup(pool, t)
 					require.NoError(t, err)
 
+					err = (&migrator.Migrator{
```

Comment:
Why do we need an explicit migration? Is it because in this test, master mode is not set? If yes, should we update that?

Reply:
Yes, this test is for specific modes like slave, where we will not be running the migration.
```
            {
				name:          "slave",
				warehouseMode: config.SlaveMode,
			},
```